misc/services/mobile-webui/mobile-webui-frontend/src/utils/qtys.js

```diff
@@ -74,3 +47,43 @@ const getDefaultDisplayPrecision = (uom, defaultPrecision = 4) => {
     return defaultPrecision;
   }
 };
+
+export const computeEffectiveValues = ({ qty, uom, precision = null }) => {
```

Comment:
IMHO `computeEffectiveValues` name is quite ambiguous.

suggestion:
* rename `formatQtyToHumanReadable` to `formatQtyToHumanReadableString`
* rename `computeEffectiveValues` to `formatQtyToHumanReadable`

wdyt?

Reply:
@teosarca the renaming sounds good. on it